plugins/infinispan100/src/main/java/org/radargun/service/InfinispanRestAPI.java

```diff
@@ -52,17 +52,13 @@ private String lookupServerIp() {
    }
 
    public CacheManagerInfo getCacheManager() {
-      // network is not reliable
-      int count = 0;
+      String url = String.format("http://%s:%s/rest/%s", serverIp, this.defaultPort, CACHE_MANAGER_RESOURCE);
```

Comment:
Should `"http://%s:%s/rest/%s"` be a constant, in case any other commands are added in this class? Not a huge deal

Reply:
I am going to drop this class and use the Infinispan10 rest client :)